community/modules/scripts/spack-install/variables.tf

```diff
@@ -92,138 +68,126 @@ variable "compilers" {
   type        = list(string)
 }
 
-variable "licenses" {
-  description = "List of software licenses to install within spack."
-  default     = null
-  type = list(object({
-    source = string
-    dest   = string
-  }))
-}
 
 variable "packages" {
   description = "Defines root packages for spack to install (in order)."
   default     = []
   type        = list(string)
 }
 
-variable "install_flags" {
```

Comment:
Do you think these might be useful still (in a later PR)? Since we are running a set of spack commands, having install flags that apply to each could make it easier for the user to write out a long list of commands that all share a flag. If so, I'll add a task to do add it.